src/tests/conformance/invalid_message.rs

```diff
@@ -0,0 +1,162 @@
+///! Contains test cases which cover ZG-CONFORMANCE-008.
+///!
+///! The node should reject the following messages post-handshake:
+///!
+///!  Version                 - Duplicate
+///!  Verack                  - Duplicate
+///!  Inv(mixed types)        - Invalid
+///!  Inv(multiple blocks)    - Invalid
+///!  Bloom filter add        - Obsolete
+///!  Bloom filter load       - Obsolete
+///!  Bloom filter clear      - Obsolete
```

Comment:
Tiny nit but more readable imo:
```suggestion
///!  Bloom filter clear      - Obsolete

```

Reply:
I'm not seeing the difference?

Reply:
Really? Could've sworn we've got files where it didn't get removed, TIL 👍